sample/SampleAspNetCoreApp/Controllers/HomeController.cs

```diff
@@ -39,25 +40,25 @@ public Task<IActionResult> Index()
 		{
 			return SafeCaptureSpan<IActionResult>(GetCaptureControllerActionAsSpanFromQueryString(),
 				"Index_span_name", "Index_span_type", async () =>
-			{

```

Comment:
Just code style... we are working on avoiding this. https://github.com/elastic/apm-agent-dotnet/pull/536